common/Environments/Models/CreateComputeSystemOperation.cs

```diff
@@ -124,19 +127,25 @@ public void StartOperation()
                 Completed?.Invoke(this, CreateComputeSystemResult);
             }
 
-            var completionStatus = EnvironmentsTelemetryStatus.Succeeded;
-
             if ((CreateComputeSystemResult == null) || (CreateComputeSystemResult.Result.Status == ProviderOperationStatus.Failure))
             {
-                completionStatus = EnvironmentsTelemetryStatus.Failed;
-                LogFailure(CreateComputeSystemResult);
+                var completionStatus = EnvironmentsTelemetryStatus.Failed;
```

Comment:
NIT: move the call to .log outside of `if`.  Technically the only item changing is the creation of `EnvironmentCreationEvent`

Reply:
removed thanks